Source/IntegrationTests/TestFiles/LitTests/LitTest/git-issues/git-issue-2064.dfy.expect

```diff
@@ -0,0 +1,35 @@
+git-issue-2064.dfy(13,9): Error: to be a refinement of abstract type 'UnitTestParent.T', type synonym 'OpaqueType.T' must support the type characteristic "no references"
+git-issue-2064.dfy(14,9): Error: to be a refinement of abstract type 'UnitTestParent.U', type synonym 'OpaqueType.U' must support the type characteristic "equality"
+git-issue-2064.dfy(15,9): Error: to be a refinement of abstract type 'UnitTestParent.V', type synonym 'OpaqueType.V' must support the type characteristic "auto-initialization"
+git-issue-2064.dfy(16,9): Error: to be a refinement of abstract type 'UnitTestParent.W', type synonym 'OpaqueType.W' must support the type characteristic "nonempty"
```

Comment:
```suggestion
git-issue-2064.dfy(16,9): Error: to be a refinement of abstract type 'UnitTestParent.W' declared `(00)`, type synonym 'OpaqueType.W' must be provably non-empty
```

Reply:
I changed it to "be nonempty". Do you think I should also include "provably" (which may suggest verification, and which may be inconsistent with the other type characteristics)? Another possibility would be "be declared nonempty". (Thinking about these possibilities, I vote for "be nonempty", but don't feel strongly and would be happy to change again, if you suggest.)

Reply:
"be nonempty" is good, since 0 could be an auto-inferred witness. We can omit the "provably" because the requirement is not a proof from the engineer's perspective, it's an expression at the declaration site (even though providing an element for a given type is like giving a proof :-) ).